includes/info.h

```diff
@@ -21,6 +21,13 @@
 #include <stdarg.h>
 #include <glib.h>
 
+enum {
```

Comment:
Maybe flags should be individual booleans -- although less efficient, it's easier to specify them in the macros individually.

Reply:
Yeah that's probably right.

Reply:
I've changed to booleans, although I was thinking of reasons to keep flags, like marking which fields (name, value, tag) should be freed.

Reply:
Yes, I've added another flag to determine if we should free the name field too -- used that to fix the Vulnerabilities thing.